src/Components/Blazor/DevServer/src/Server/Program.cs

```diff
@@ -18,12 +25,24 @@ public class Program
         /// <summary>
         /// Intended for framework test use only.
         /// </summary>
-        public static IWebHost BuildWebHost(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseConfiguration(new ConfigurationBuilder()
-                    .AddCommandLine(args)
-                    .Build())
-                .UseStartup<Startup>()
-                .Build();
+        public static IHost BuildWebHost(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureHostConfiguration(cb => {
+                    var applicationPath = args.SkipWhile(a => a != "--applicationpath").Skip(1).FirstOrDefault();
+                    var name = Path.ChangeExtension(applicationPath,".StaticWebAssets.xml");
```

Comment:
We compute the value like this today. We'll pass it in through MSBuild in the future.

Reply:
Would be good to have a tracking issue for the cleanup work